verify.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require_once 'functions.php';
+if((!isset($_POST['charname']) || $_POST['charname'] === '') 
+    || (!isset($_POST['class']) || $_POST['class'] === '')
+    || (!isset($_POST['level']) || $_POST['level'] === '')
+    || (!isset($_POST['strength']) || $_POST['strength'] === '')
+    || (!isset($_POST['dexterity']) || $_POST['dexterity'] === '')
+    || (!isset($_POST['constitution']) || $_POST['constitution'] === '')
+    || (!isset($_POST['intelligence']) || $_POST['intelligence'] === '') 
+    || (!isset($_POST['wisdom']) || $_POST['wisdom'] === '') 
+    || (!isset($_POST['charisma']) || $_POST['charisma'] === '')) {
```

Comment:
I would suggest moving this conditional into it's own function.

The idea would be that you can call the function, pass in $_POST as a param, and it then runs this big chunky check and returns true/false depending on whether all of the data is present or not. 

That way you could re-use it, possibly even for edit functionality!

Reply:
Just to clarify, the main reason I suggest passing in $_POST as a param is because then you could write some nice unit tests for the function!

Whereas if the function accessed $_POST itself, there would be no way to pass data into it inside of a unit test